src/generator/AutoRest.Ruby.Azure/Templates/AzureModelTemplate.cshtml

```diff
@@ -0,0 +1,69 @@
+﻿@using System
```

Comment:
Looked into other options. For now, since we are using cshtml files, we do not have anyother option. No changes here